src/unstract/sdk/tool/base.py

```diff
@@ -38,6 +40,29 @@ def __init__(self, log_level: LogLevel = LogLevel.INFO) -> None:
         self.execution_id = ""
         self.org_id = ""
         self._exec_metadata = {}
+        self.filestorage_provider = None
+        self.workflow_filestorage = None
+        self.execution_dir = None
+        filestorage_provider_env = os.environ.get(
+            ToolEnv.WORKFLOW_EXECUTION_FS_PROVIDER
+        )
+        if filestorage_provider_env:
+            self.execution_dir = Path(self.get_env_or_die(ToolEnv.EXECUTION_DATA_DIR))
```

Comment:
@muhammad-ali-e I think this needs to be done in the next set...

Reply:
This is not for the backend application; it's specifically for tool execution (tool's base class).
- This was previously defined as `DATA_DIR` in the SDK (retained in the existing code for backward compatibility). In our version (FileStorage), it has been renamed to `EXECUTION_DATA_DIR`.
- From my understanding, the reason we kept `DATA_DIR` in the SDK (with the same name) is related to the flow: it gets passed from the runner when the container is spawned (`run_container`). Therefore, it needs to remain in the tool's environment without changing the environment variable's name.
- If we were to initialize this in tools instead, every tool would need to implement the same logic, adding redundancy.

Reply:
Let us discuss this @muhammad-ali-e . I think it is find for the env to be   part of the tool. But adding env getter logic in SDK is something we need to discuss on. Even if the logic becomes redundant, I think it is still ok. It will give the flexibility to the tool as to how to maintain the env and whether to even have the env or not for whatsoever reasons.